README.md

```diff
@@ -192,6 +192,13 @@ To preserve notebook metadata, such as language version, use:
 nb-clean add-filter --preserve-notebook-metadata
 ```
 
+By default, `nb-clean` will not delete all notebook metadata. To completely
```

Comment:
GitHub doesn't allow commenting on lines not included within the diff in a PR, but we also need to mention `--remove-all-notebook-metadata` where we list the available options in the "Checking" and "Cleaning (interactive)" sections above, starting at lines 61 and 116. In those, we should also make clear that `--preserve-notebook-metadata` only checks for the presence of the `language_info.version` key, to explain how it differs from `--remove-all-notebook-metadata`.